system/sharedClasses/playlist.xml

```diff
@@ -185,7 +185,10 @@
 			<variation>Mode110</variation>
 			<variation>Mode610</variation>
 		</variations>
-		<command>$(SYSTEMTEST_CMD_TEMPLATE) -test=SharedClasses -test-args=$(Q)sharedClassMode=SCM01,sharedClassTest=MultiThread$(Q); \
+		<!-- Some platforms may have max user processes too low by default for this test
+		 - e.g. see https://github.com/AdoptOpenJDK/openjdk-infrastructure/issues/1851 -->
+		<command>if [[ ! $(Q)$(PLATFORM)$(Q) == *$(Q)win$(Q)* ]] ; then ulimit -u 2048 $(AND_IF_SUCCESS) echo ulimit -u set to 2048; ulimit -a; fi; \
+		$(SYSTEMTEST_CMD_TEMPLATE) -test=SharedClasses -test-args=$(Q)sharedClassMode=SCM01,sharedClassTest=MultiThread$(Q); \
```

Comment:
Since it's only the "max user processes" field that's being updated here, can we just output `ulimit -u`, instead of the full `ulimit -a` here? Since we are already printing out `ulimit -a` via MachineInfo, printing the whole `ulimit -a` output again seems to increase the job output text unnecessarily. 

In other words, can we do something along the line of the following? :- 

```
echo "ulimit -u set to: `ulimit -u`"
```

Reply:
I will make the change and retest.